packages/lib/pcd-types/src/pcd.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
There are lots of `any`s in this file and I don't feel like I understand them well enough to change them.